bin/installDB.bash

```diff
@@ -255,7 +255,7 @@ error_check "Error starting farmOS."
 echo "  Started."
 
 echo "Clearing the Drupal cache..."
-clearDrupalCache.bash
+docker exec fd2_farmos drush cr
```

Comment:
This had to be done as the host machine does not recognize the file path at clearDrupalCache.bash